tests/unit/controllers/missed-appointments.js

```diff
@@ -51,23 +50,23 @@ exports['get returns zero if all registrations have delivered'] = function(test)
   var fti = sinon.stub(db, 'fti');
   fti.onFirstCall().callsArgWith(2, null, {
     rows: [
-      { 
-        doc: { 
```

Comment:
this lot doesn't seem to be relevant.  if you really want to make this change, just push to `master` IMO